src/ring_theory/perfection.lean

```diff
@@ -96,6 +90,10 @@ lemma coeff_frobenius (f : ring.perfection R p) (n : ℕ) :
   coeff R p (n + 1) (frobenius _ p f) = coeff R p n f :=
 by convert coeff_pow_p f n
 
+lemma coeff_iterate_frobenius (f : ring.perfection R p) (n m : ℕ) :
```

Comment:
Would you mind adding a `'` version that uses nat-subtraction on the RHS (and assumes `m \le n`).

Reply:
Done.